sycl/test-e2e/bindless_images/vulkan_interop/sampled_images.cpp

```diff
@@ -231,15 +334,58 @@ bool run_test() {
   printString("Getting memory file descriptors and calling into SYCL\n");
   // Pass memory to SYCL for modification
   auto input_fd = vkutil::getMemoryOpaqueFD(inputMemory);
-  bool validated = run_sycl(input_fd, width, height);
+  bool result = run_sycl<NDims, DType, NChannels, CType, COrder, KernelName>(
+      dims, localSize, input_fd);
 
   // Cleanup
   vkDestroyBuffer(vk_device, inputStagingBuffer, nullptr);
   vkDestroyImage(vk_device, inputImage, nullptr);
   vkFreeMemory(vk_device, inputStagingMemory, nullptr);
   vkFreeMemory(vk_device, inputMemory, nullptr);
 
-  return validated;
+  return result;
+}
+
+bool run_tests() {
+  bool valid = run_test<2, float, 4, sycl::image_channel_type::fp32,
+                        sycl::image_channel_order::rgba, class float_2d>(
+      {16, 16}, {2, 2}, 0);
+
+  valid &= run_test<2, float, 2, sycl::image_channel_type::fp32,
+                    sycl::image_channel_order::rg, class float_2d_large>(
+      {1024, 1024}, {4, 2}, 0);
+
+  valid &= run_test<3, char, 2, sycl::image_channel_type::signed_int8,
+                    sycl::image_channel_order::rg, class float_3d>(
+      {256, 16, 2}, {2, 2, 2}, 0);
+
+  valid &= run_test<2, uint32_t, 1, sycl::image_channel_type::unsigned_int32,
+                    sycl::image_channel_order::r, class uint32_2d>({64, 32},
+                                                                   {4, 2}, 0);
+
+  valid &= run_test<3, uint32_t, 4, sycl::image_channel_type::unsigned_int32,
+                    sycl::image_channel_order::rgba, class uint_3d_large>(
+      {1024, 256, 16}, {2, 2, 4}, 0);
+
+  valid &= run_test<2, int32_t, 1, sycl::image_channel_type::signed_int32,
+                    sycl::image_channel_order::r, class int32_2d>({64, 32},
+                                                                  {4, 2}, 0);
+
+  valid &= run_test<3, int32_t, 2, sycl::image_channel_type::signed_int32,
+                    sycl::image_channel_order::rg, class int32_3d>(
+      {64, 32, 64}, {4, 2, 4}, 0);
+
+  valid &= run_test<3, int16_t, 1, sycl::image_channel_type::signed_int16,
+                    sycl::image_channel_order::r, class int16_3d>({64, 32, 64},
+                                                                  {4, 2, 4}, 0);
+
+  if (!valid) {
+    std::cout << "sampled images test has failure(s)\n";
+  } else {
+    std::cout << "sampled images test passes\n";
+  }
```

Comment:
Why the switch from a simple `Test passed!` output to this? Or even `All tests passed!` as in our other consolidated testing?

If you decide to change this then can you also edit the `unsampled_images.cpp` file to print the same message.

Reply:
done.